From 7cebf964dbe7e3b3ce886d53f642ea4b6f8d3db8 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 24 Dec 2018 20:34:30 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-sapphire-excavator-linux: add
 cec pinctrl

GPIO4_C7 is used for hdmi-cec since rk3399-sapphire-excavator-v12.

Change-Id: I8c26debdcb1827e067b7a95acf6151160b61e03d
Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
index 31c1dc8ac265..4ce75841eb8a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
@@ -296,8 +296,8 @@
 };
 
 &hdmi {
-	/* remove the hdmi_cec, reused by edp_hpd */
-	pinctrl-0 = <&hdmi_i2c_xfer>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&hdmi_i2c_xfer>, <&hdmi_cec>;
 	#address-cells = <1>;
 	#size-cells = <0>;
 	#sound-dai-cells = <0>;
-- 
2.35.3

